src/matter/cluster/BasicInformationCluster.ts

```diff
@@ -59,15 +66,15 @@ export const BasicInformationCluster = Cluster({
         location: WritableAttribute(6, StringT({ length: 2 }), { default: "XX", writeAcl: AccessLevel.Administer } ),
 
         /** Version number of the hardware of the Node. The meaning of its value, and the versioning scheme, are vendor defined. */
-        hardwareVersion: Attribute(7, UnsignedIntT, { default: 0 }),
+        hardwareVersion: Attribute(7, BoundedUnsignedIntT({ min: 0, max: 0xFFFF }), { default: 0 }),
 
-        /** Human readable representation of the {@link BasicInformationCluster.attributes.hardwareVersion hardwareVersion} attribute. */
+        /** Human-readable representation of the {@link BasicInformationCluster.attributes.hardwareVersion hardwareVersion} attribute. */
         hardwareVersionString: Attribute(8, StringT({ minLength: 1, maxLength: 64 })),
 
         /** Current version number for the software running on this Node. A larger value is newer than a lower value. */
         softwareVersion: Attribute(9, UnsignedIntT, { default: 0 }),
 
-        /** Human readable representation of the {@link BasicInformationCluster.attributes.softwareVersion softwareVersion} attribute. */
+        /** Human-readable representation of the {@link BasicInformationCluster.attributes.softwareVersion softwareVersion} attribute. */
```

Comment:
But I think slowly we shpuld thik about maximum text length! Else these "one line comments" can get very strange".

Reply:
Let's worry about maximum text length per line when we add lint.
Using one line Jsdoc makes the code quite readable but I agree @link makes the line longer...